pkgs/development/python-modules/pylibgen/default.nix

```diff
@@ -0,0 +1,25 @@
+{ pythonPackages, fetchPypi , python , stdenv }:
```

Comment:
no `pythonPackages` in `python-modules`. This applies to other expressions as well.